ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml

```diff
@@ -100,11 +91,16 @@ StackLayout {
                     }
                 }
 
+                Item { Layout.fillWidth: true }
+
                 StatusButton {
+                    Layout.preferredHeight: 38
+                    Layout.alignment: Qt.AlignTop
                     objectName: "communityOverviewSettingsEditCommunityButton"
                     visible: root.editable
                     text: qsTr("Edit Community")
                     onClicked: root.currentIndex = 1
+                    size: StatusBaseButton.Size.Small
```

Comment:
Seems a bit awkward to hardcode both the `size` and `height`

Reply:
The size helps in setting the appropriate font size, the height is there to match height from design.